src/data/emacs-mode/agda-input.el

```diff
@@ -661,19 +734,29 @@ order for the change to take effect."
   ("\\"         . ("\\"))
   ("en"         . ("–"))
   ("em"         . ("—"))
+  ("!"          . ("！"))
   ("!!"         . ("‼"))
+  ("?"          . ("？"))
   ("??"         . ("⁇"))
   ("?!"         . ("‽" "⁈"))
   ("!?"         . ("⁉"))
   ("die"        . ,(agda-input-to-string-list "⚀⚁⚂⚃⚄⚅"))
-  ("asterisk"   . ,(agda-input-to-string-list "⁎⁑⁂✢✣✤✥✱✲✳✺✻✼✽❃❉❊❋"))
+  ("asterisk"   . ,(agda-input-to-string-list "⁎⁑⁂✢✣✤✥✱✲✳✺✻✼✽❃❉❊❋＊"))
   ("8<"         . ("✂" "✄"))
   ("tie"        . ("⁀"))
   ("undertie"   . ("‿"))
   ("apl"        . ,(agda-input-to-string-list "⌶⌷⌸⌹⌺⌻⌼⌽⌾⌿⍀⍁⍂⍃⍄⍅⍆⍇⍈
                                                ⍉⍊⍋⍌⍍⍎⍏⍐⍑⍒⍓⍔⍕⍖⍗⍘⍙⍚⍛
                                                ⍜⍝⍞⍟⍠⍡⍢⍣⍤⍥⍦⍧⍨⍩⍪⍫⍬⍭⍮
                                                ⍯⍰⍱⍲⍳⍴⍵⍶⍷⍸⍹⍺⎕"))
+  ("#"          . ("＃"))
+  ("%"          . ("％"))
+  ("&"          . ("＆"))
+  ("*"          . ("＊"))
+  ("/"          . ("／＼"))
```

Comment:
Is `\/` a good way to type something that looks like `/\`?

Reply:
Oops, I meant for this to be a list of two characters, the fullwidth slash and the fullwidth backslash.

Reply:
I think  `\/` clearly makes sense for the fullwidth slash.  I wasn't sure about the fullwidth backslash, but putting it on `\\` seemed like a bad idea.

Reply:
OK.